__tests__/index.ts

```diff
@@ -139,7 +139,7 @@ function createTest({
  */
 const constantizeLogEntryForTest = logEntry => {
   const makePlaceholder = (key, placeholder = undefined) => {
-    if (logEntry[key]) {
+    if (key in logEntry) {
```

Comment:
Bug: `logEntry[key]` will return `false` if the dynamic value is `0`. This is not desirable; we want to replace all dynamic values, even `0`. :smile: